i3_agenda/__init__.py

```diff
@@ -121,20 +123,23 @@ def main():
     event = datetime.datetime.fromtimestamp(closest.unix_time)
     today = datetime.datetime.today()
     tomorrow = today + datetime.timedelta(days=1)
+    next_week = today + datetime.timedelta(days=7)
     urgent = today + datetime.timedelta(minutes=5)
 
     result = str(get_display(closest.summary))
 
-    if args.limchar >= 0 and len(result) > args.limchar:
+    if 0 <= args.limchar < len(result):
```

Comment:
switch it so the variable will be before the comparison  
ie: `x >= 0` instead of `0 <= x`

Reply:
it is suggested by the IDE, I can "simplify changed comparison", I can revert it if you like.

Reply:
Oh you're right, I didn't notice you removed the `and` in my first read.